nixos/doc/manual/development/writing-nixos-tests.xml

```diff
@@ -262,8 +262,14 @@ startAll;
     <literal>waitForWindow(qr/Terminal/)</literal>.</para></listitem>
   </varlistentry>
 
+  <varlistentry>
+    <term><methodname>copyFileFromHost</methodname></term>
+    <listitem><para>Copies a file from host to machine, e.g.,
+    <literal>copyFileFromHost("myfile", "/etc/my/important/file")</literal>.</para></listitem>
```

Comment:
Can you add more about the params, like the left param is the source file on the host, relative to what, and the second file is the destination on the VM?